.goreleaser.yml

```diff
@@ -31,9 +45,6 @@ gomod:
 nfpms:
  - id: go-ssb-room
    package_name: go-ssb-room
-   replacements:
```

Comment:
i removed these replacements because they seem unnecessary, i'd argue the default names are more clear because they match everything else. but happy to revert this.

Reply:
Seems legit, let's use the defaults.